llvm/test/tools/llvm-objcopy/ELF/compress-sections-within-segment.s

```diff
@@ -0,0 +1,58 @@
+## Disallow (de)compression for sections within a segment as they are
```

Comment:
Is it worth a test case demonstrating the behaviour when you specify a section in a segment with a compression type that matches what it has already? I.e. it shows that llvm-objcopy is "dumb" and doesn't try to figure out whether we actually need to do anything before emitting the error.

Reply:
Thanks for the suggestion. Added